service/history/replication/stream_receiver.go

```diff
@@ -191,11 +199,11 @@ func (r *StreamReceiverImpl) recvEventLoop() error {
 
 func (r *StreamReceiverImpl) ackMessage(
```

Comment:
Can you make a note on this function that it returns the watermark info if present?